<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ComboBoxDemo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/icons/aim.ico</normaloff>:/icons/icons/aim.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>简单的ComboBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnInitItems">
          <property name="text">
           <string>初始化列表</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="chkEditable">
          <property name="text">
           <string>可编辑</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QComboBox" name="comboItems">
          <item>
           <property name="text">
            <string>杭州</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/icons/UNIT.ICO</normaloff>:/icons/icons/UNIT.ICO</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>南昌</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/icons/UNIT.ICO</normaloff>:/icons/icons/UNIT.ICO</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnClearItems">
          <property name="text">
           <string>清空列表</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>有用户数据的ComboBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnInitCities">
          <property name="text">
           <string>初始化城市+区号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboCities">
          <property name="placeholderText">
           <string>请选择一个城市</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>清空文本</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
